### DISCUSSION TITLE
Draft status for discussions

### DISCUSSION THREAD

**Discussion by Relequestual** [community/community#3989] [Discussions]
I was writing a new discussion and had to take pause and shift to something else.
I later had a kernel panic... those hours crafting that discussion intro are gone.
If I'd have had the option to save the discussion as a draft, I would have done so.
I would expect drafts to not allow commenting and not send a "new discussion" webhook event.

We're using webhooks to post new discussions and comments in our slack server.

**Comment by peerkleio** [comment]
Similar experience here, though I have not lost a discussion yet. I tend to frequently copy the discussion information to another document somewhere just in case of a browser or system crash.

**Comment by rafaeldjpbrochado** [comment]
Great suggestion!

**Comment by philip-gai** [comment]
The [Announcement Drafter GitHub App](https://github.com/marketplace/repo-st) solves this problem, draft announcements or any type of discussion using PRs and they'll get created when the PR is merged! 🚀

**Comment by sethfri** [comment]
Having just lost about 6 paragraphs of a discussion post I was writing because of a laptop restart, I would really, really like this feature

**Comment by ChrisAcrobat** [comment]
This would be an awesome feature.
>I would expect drafts to not allow commenting and not send a "new discussion" webhook event.

I would also like to add read and write privileges to teams and members while in draft-mode. Team 1 could write and Team 2 read and approve. Post the new discussion could be looked to the author, but a separata send ("Start discussion") privilege could also be nice.

@philip-gai I will definitely look into [Announcement Drafter GitHub App](https://github.com/marketplace/repo-st), thanks for the tip.

**Comment by outragedhuman** [comment]
I would like to see this feature too.
